src/Ord.ts

```diff
@@ -181,3 +181,9 @@ export const getProductOrd = <A, B>(OA: Ord<A>, OB: Ord<B>): Ord<[A, B]> => {
 export const getDualOrd = <A>(O: Ord<A>): Ord<A> => {
   return fromCompare((x, y) => O.compare(y, x))
 }
+
+/**
+ * @Instance
```

Comment:
@mlegenhausen thanks for this PR, a nitpick: could you please change this to lowercase (`@instance`)?